Cargo.toml

```diff
@@ -10,6 +10,8 @@ repository = "https://github.com/nagisa/rust_libloading/"
 documentation = "https://nagisa.github.io/rust_libloading/"
 
 [dependencies]
+lazy_static = "0.2"
+
+[target.'cfg(windows)'.dependencies]
```

Comment:
Yeah, I feel like maintaining the support for, say, 1.6 is somewhat more desirable than avoiding a download and parse of winapi.

I think I’d be fine with taking something like this a few more versions of Rust later. Please fill an issue/suggest the change in a separate PR, lest I forget about it.